docs/source/features/tool_calling.md

```diff
@@ -91,6 +91,12 @@ For best results, we recommend ensuring that the expected output format / schema
 To use a named function, you need to define the functions in the `tools` parameter of the chat completion request, and
 specify the `name` of one of the tools in the `tool_choice` parameter of the chat completion request.
 
+## Required Function Calling
+
+vLLM supports the `tool_choice='required'` option in the chat completion API. Similar to the named function calling, it also uses guided decoding, so this is enabled by default and will work with any model.
```

Comment:
> will work with any model

Is there any guarantee? If not, we should remove this claim?

Reply:
Used the same wording with "supported models" just like named function calling.